tailwind.config.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
   content: ["./static/views/**/*.html", "./src/app/**/*.{js,ts,jsx,tsx}"],
   plugins: [require("@tailwindcss/forms"), require("@tailwindcss/line-clamp")],
   theme: {
+    screens: {
```

Comment:
I don't think we need a new breakpoint in order to limit the width of a component.

Reply:
OK, I will look at the relevant code of # 1955，to see the effect of the solution

Reply:
![WX20230116-103840](https://user-images.githubusercontent.com/3917994/212586880-efe48426-d83e-4fd0-8240-b5491c3b2e09.png)
Just like mobile adaptation, pop view is more like a small-sized mobile page, and the current breakpoint cannot distinguish whether it is running on a smaller-sized page

@rolznz Do you have any ideas, I think this kind of small size problem will be quite common in the future

Reply:
@reneaaron @qqqzhch I made a new PR here https://github.com/getAlby/lightning-browser-extension/pull/1982 which doesn't introduce a breakpoint. I am not really happy with the solution (it feels a bit hacky), but it is isolated to the account menu.